jasmine-standalone-3.6.0 (1) 2/spec/BowlingSpec.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+describe('Bowling', function() {
+  var score;
+  var pins;
+
+  it('starts at 0 score', function() {
+    score = new Bowling();
```

Comment:
Since this is repeated in every section, perhaps  set a beforeEach{} right after the var(at the top) so that it isn't repeated in every test :).